housingbayarea/bloom
# 619/redirect to doorway

# Pull Request Template

## Issue Overview

This PR addresses #619 


## Description

These changes utilize session storage to capture if an applicant is coming from Doorway.
Two things to note:
1) Doorway would have to append "&doorway=true" when linking to our applications
2) I'm unsure if the production url has been determined for doorway but that will need to be included in these changes before merging. I have the dev link in there now for testing. Do you know if they have decided this @ludtkemorgan?

## How Can This Be Tested/Reviewed?
Pull this down locally
1. Find a listing using the common app and click apply online
2. When on the choose-language page, add "&doorway=true" to the url, and click enter
3. Then fill out the application and you will realize that the "Browse more listings" link on the confirmation page shows the updated language and redirects back to doorway.

4. Repeat this process on a different listing/application but without the appended doorway on the url
5.  You will notice that the confirmation page is unchanged compared to the current state of prod.
## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
